Makefile

```diff
@@ -87,6 +87,13 @@ mypy: regen-metaparser
 black:
 	black pegen tatsu test scripts
 
+format-pegen:
```

Comment:
The difference in naming style for this target from ‘black’ somehow bothers me.

Reply:
Should I rename `black` to `format-python` and `format-pegen` to `format-c`?

Reply:
Sounds good.